src/lib/rest/CMakeLists.txt

```diff
@@ -46,7 +47,8 @@ SET (HEADERS
     orionLogReply.h
     OrionError.h
     HttpStatusCode.h
-    StringFilter.h    
```

Comment:
If StringFilter.h is not needed here, maybe StringFilter.cpp is not needed in line 33?

Reply:
Not sure why this line was removed ...

Reply:
Seems like a bug in github ...
I am editing `src/lib/rest/CMakeLists.txt` right now and `StringFilter.h` is present ...
Let's see what happens after a new commit ...

NTC, for now

Reply:
Ah, now I understand.
Look at line 50 ...
For some reason StringFilter.h is marked as removed and inserted
Perhaps I changed a TAB for spaces or something when I added restServiceLookup.h ...

NTC
￼
￼